tasties_app/views.py

```diff
@@ -96,3 +97,40 @@ def view_recipe(request, recipe_id=None):
     categories = recipe.categories.all()
     context = {'recipe': recipe, 'ingredients': ingredients, 'rating': rating, 'categories': categories}
     return render(request, 'tasties_app/view_recipe.html', context)
+
+
+@login_required(login_url='login')
+def create_recipe(request):
+    recipe = Recipe(author_id=request.user)
+    IngredientFormSet = inlineformset_factory(Recipe,
+                                              Ingredient,
+                                              fields=('description', 'measurement_unit', 'amount'),
+                                              min_num=1,
+                                              validate_min=True,
+                                              extra=9)
+
+    if request.method == 'POST':
+        recipe_form = CreateRecipeForm(request.POST, request.FILES, instance=recipe)
+        ingredient_formset = IngredientFormSet(request.POST, instance=recipe)
```

Comment:
Maybe we can change the name a bit, it took me some time to understand what is `formset`...
WDYT about `ingredient_form_set`?

Reply:
`formset` is the common and well known name for this Django feature, as it appears [here](https://docs.djangoproject.com/en/4.1/topics/forms/formsets/#:~:text=A-,formset,-is%20a%20layer).

Reply:
ok, thanks for the clarification